ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/recovery/ActionTestClient.java

```diff
@@ -303,7 +303,11 @@ private static boolean test_setup()
         boolean setupOk = false;
 
         try {
-            String host = InetAddress.getLocalHost().getHostAddress();
+            System.out.println("Here");
```

Comment:
The stack trace provides the same information as these output statements so it's safe to remove them.

Reply:
I will remove them, one of them prints out the host but I think it might not be necessary to isolate this particular issue as it fails before it is printed